changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 12th September 2023
+
+* Removed 'AccountingCounter' as a discriminator from [Counter Type Discriminator](../operations/counters.md#counter-type-discriminator) since it would never get returned nor it exists in our database.
```

Comment:
```suggestion
* Removed `AccountingCounter` from [Counter Type Discriminator](../operations/counters.md#counter-type-discriminator) since it is never used.
```